lib/bin/lwm2m_carrier/doc/requirements.rst

```diff
@@ -25,13 +25,19 @@ Below are some of the requirements and limitations of the application while runn
 * The LwM2M carrier library controls the LTE link.
 
    * This is needed for storing keys to the modem, which requires disconnecting from an LTE link and connecting to it.
-   * The application should not connect to the LTE link between a :c:macro:`LWM2M_CARRIER_EVENT_DISCONNECTING` event and :c:macro:`LWM2M_CARRIER_EVENT_CONNECTED` event.
+   * The application cannot use the LTE link between a :c:macro:`LWM2M_CARRIER_EVENT_DISCONNECTING` event and :c:macro:`LWM2M_CARRIER_EVENT_CONNECTED` event.
+     Instead, the application must wait for the :c:macro:`LWM2M_CARRIER_EVENT_LTE_READY` event.
```

Comment:
I think all these lines can be summed up as:
`The application must wait for the LWM2M_CARRIER_EVENT_LTE_READY event before using the LTE link`.

Reply:
I do believe the registered event etc was added so that you would get an event sooner that would indicate to application that you can in fact go online sooner, what say you @stig-bjorlykke